Sources/Contentful/Decodable.swift

```diff
@@ -93,6 +93,19 @@ internal extension Decodable where Self: AssetDecodable {
     }
 }
 
+internal extension Decodable where Self: Node {
+    static func popNodeDecodable(from container: inout UnkeyedDecodingContainer) throws -> Self {
+        do {
+            let contentDecodable = try container.decode(self)
+            return contentDecodable
+
+        } catch let error {
+            print(error)
```

Comment:
Should this print be removed? Seems like a debugging statement

Reply:
ooo good catch, definitely needs to be removed